core/src/main/java/io/hyperfoil/core/steps/ScheduleDelayStep.java

```diff
@@ -62,7 +62,7 @@ public boolean invoke(Session session) {
       long delay = blockedUntil.timestamp - now;
       if (delay > 0) {
          log.trace("Scheduling #{} to run in {}", session.uniqueId(), delay);
-         session.executor().schedule(session.runTask(), delay, TimeUnit.MILLISECONDS);
+         session.executor().schedule(session.runTask(), delay, TimeUnit.NANOSECONDS);
```

Comment:
This is not saving the ScheduledFuture nowhere - it is just fixing the ns granularity right?

Reply:
Yeah correct, this is just about the `ns` granularity